.github/workflows/update_python_dependencies.yml

```
@@ -0,0 +1,38 @@
+name: Update Python Dependencies
+
+on:
+    workflow_dispatch:
```

±: Two changes:

```suggestion
    workflow_call:
        inputs:
            python-version:
                description: "Python version used by pip-tools."
                required: false
                type: string
                default: '3.10'
```

1. Use `workflow_call` in the callee, `workflow_dispatch` for manually triggering a workflow.
2. Because the web apps can be on different Python versions we need to allow them to specify the version to use.